### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by W0naN0w**

### CODE REVIEW

File: docs/nmap-install.xml [hunk 1]
```diff
@@ -1008,9 +1008,7 @@ command such as <command>nmap scanme.nmap.org</command> from any directory.</par
 <indexterm class="startofrange" id="inst-macosx-indexterm"><primary>Mac OS X</primary></indexterm>
 <indexterm><primary>Apple Mac OS X</primary><see>Mac OS X</see></indexterm>
 
-<para>Nmap has supported Mac OS X since 2001, and our support has only improved over time.  While Mac users can compile Nmap themselves, we also offer an executable installer.  Nmap
-is also available through systems such as MacPorts and Fink which
-package Unix software for Mac OS X.</para>
+<para>Nmap has supported Mac OS X since 2001, and our support has only improved over time. While Mac users can compile Nmap themselves, we also offer an executable installer. Nmap makes use of Jhbuild and gtk-mac-bundler which are used to build other projects for Mac OS X, such as OpenSSL, libapr, libsvn... Nmap is also available through systems such as MacPorts and Fink which package Unix software for Mac OS X.</para>
```
Comment (dmiller-nmap) [inline docs/nmap-install.xml]: Please wrap paragraphs. Makes it easier to proofread.

**Comment by W0naN0w** [comment]
Since I am not allowed to modify images in the docs, I am going to post here the new screenshots I photoshopped that would serves as new figures in the Apple Mac OS X section (i.e. "Chapter 2. Obtaining, Compiling, Installing, and Removing Nmap") :

- Figure 2.2 : 
image

- Figure 2.3 :
image

- Figure 2.4 :
image

- New Figure 2.5, illustrating point number 3. of "Executable Installer" :
image

Waiting for feedback!

**Comment by W0naN0w** [comment]
Thank you @cldrn and @dmiller-nmap for your comments. I went through all the files located in the `docs/` folder and reviewed the work I've done during this summer. The only thing I've found that still needs to be corrected is the French accents used in the Nmap man (fr version : nmap-man-fr.xml). I currently have this kind of problems when doing `man nmap`:

Also, I think that's an encoding issue, we may have to change from `latin1` to `utf8`... what do you think? I've tried to `configure && make && make install`, but it doesn't seems to change the `nmap-fr.1` file in `docs/`.

@dmiller-nmap do you think there is anything else I can document?

**Comment by W0naN0w** [comment]
These modifications in the documentation have been uploaded to the SVN trunk. Thanks to @dmiller-nmap for replacing the images posted in this PR.

**Pull request closed without merging**